packages/react-network/src/server.ts

```diff
@@ -22,5 +22,14 @@ export function applyToContext<T extends Context>(
     ctx.set(header, value);
   }
 
+  for (const [cookie, options] of cookies) {
+    const {value, ...rest} = options;
+    ctx.cookies.set(cookie, value, rest);
+  }
+
+  // eslint-disable-next-line no-warning-comments
+  // TODO: Add a watcher function that reapplies the cookies
```

Comment:
If we just used https://www.npmjs.com/package/universal-cookie it would handle updating the cookies for us